config/routes.rb

```diff
@@ -0,0 +1,3 @@
+UiComponents::Engine.routes.draw do
```

Comment:
Okay, but then this file (and many others) are wrong here and belong in the dummy app, right? This exposes all the styleguide displaying logic to the target app.